core/src/main/java/org/radargun/stats/Statistics.java

```diff
@@ -51,6 +51,12 @@
     * Mark this moment as start of the measurement.
     * No operations should be recorded before this call.
     */
+   /**
+    * TODO
+    * We are using TimeService.currentTimeMillis() or TimeService.nanoTime()
```

Comment:
Begin and end are wall clock times that should tell you when was the test running/when were the statistics collected.
Note that `nanoTime()` is unsuitable for measuring extended periods of time as it can skew with CPU throttling and such.